org.kde.kdenlive.json

```diff
@@ -0,0 +1,448 @@
+                }
+            ]
+        },
+        {
+            "name": "rtaudio",
+            "config-opts": [
+                "--disable-static",
+                "--enable-shared"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.music.mcgill.ca/~gary/rtaudio/release/rtaudio-5.0.0.tar.gz",
+                    "sha256": "799deae1192da52cc2c15a078ed3b42449580be7d096fe9bc841c5bba0289c57"
+                }
+            ]
+        },
+        {
+            "name": "ffmpeg",
+            "config-opts": [
+                "--enable-gpl",
+                "--disable-static",
+                "--enable-shared",
+                "--disable-ffserver",
+                "--disable-doc",
+                "--disable-alsa",
+                "--enable-libfontconfig",
+                "--enable-libfreetype",
+                "--enable-libopenjpeg",
+                "--enable-libopus",
+                "--enable-libpulse",
+                "--enable-librsvg",
+                "--enable-libvidstab",
+                "--enable-libvpx",
+                "--enable-libx264",
+                "--enable-libx265",
+                "--enable-libxml2"
+            ],
+            "cleanup": [
+                "/share/ffmpeg/examples"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ffmpeg.org/releases/ffmpeg-3.4.1.tar.xz",
+                    "sha256": "5a77278a63741efa74e26bf197b9bb09ac6381b9757391b922407210f0f991c0"
+                }
+            ]
+        },
+        {
+            "name": "opencv",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_LIBDIR=lib",
+                "-DBUILD_opencv_xfeatures2d=OFF"
+            ],
+            "cleanup": [
+                "/bin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/opencv/opencv/archive/3.4.0.tar.gz",
+                    "sha256": "678cc3d2d1b3464b512b084a8cca1fad7de207c7abdf2caa1fed636c13e916da"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/opencv/opencv_contrib/archive/3.4.0.tar.gz",
+                    "sha256": "699ab3eee7922fbd3e8f98c68e6d16a1d453b20ef364e76172e56466dc9c16cd",
+                    "dest-filename": "opencv_contrib.tar.gz"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "tar xf opencv_contrib.tar.gz --strip 1"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "gavl",
```

Comment:
this needs the autoreconf dance as well. There also seems to be a slightly different compile error here for ARM

Reply:
Ah, I remember now - I saw the same issue in Pitivi which also uses gavl. It's something to do with libm. If you can work out how to fix it, all the better! If not, then it's OK to use a flathub.json file to ban the arm architecture for now.